channels/stable-4.3.yaml

```diff
@@ -1,9 +1,27 @@
 name: stable-4.3
 versions:
-# 4.2.16+amd64 - withheld as it's only valid into 4.3.0,4.3.1 people should go direct to 4.3.5
+- 4.2.0
+- 4.2.1
+- 4.2.2
+# no 4.2.3 because it was rolled into 4.2.4
+- 4.2.4
+# no 4.2.5 because it was cut missing some intended update sources
+# no 4.2.6 because it pulled in CI-built images due to due to release job modification for multi-arch
+- 4.2.7
+- 4.2.8
+- 4.2.9
+- 4.2.10
+# - 4.2.11 failed to run tests, we never officially released it, but we accidentally
+# put it in a channel! (same for s390x)   Now we shouldn't pull it, just in case
+# someone is on it
+- 4.2.11
+- 4.2.12
+- 4.2.13
+- 4.2.14+amd64
+- 4.2.16+amd64
 # not 4.2.17 because we had a long quiet time after 4.2.16 with no releases
```

Comment:
We have two of these lines in this file, one from 9f94edc3 and another from 505f7a64.  Can you remove the line further down in this file with this PR?  Or I can file a follow-up PR removing the duplicate line after this PR lands.

Reply:
ack